models/contacts.js

```diff
@@ -1,19 +1,58 @@
-// const fs = require('fs/promises')
+const fs = require('fs');
+const path = require('path')
+const contactsDirectory = path.join(__dirname, 'contacts.json')
+const Joi = require('joi')
 
-const listContacts = async () => {}
-
-const getContactById = async (contactId) => {}
+function listContacts() {
+  const contactsData = fs.readFileSync(contactsDirectory);
+  const contacts = JSON.parse(contactsData);
+  return contacts;
```

Comment:
```suggestion
  return JSON.parse(contactsData);
  
``` в таких випадках можна трохи скорочувати